coding-guides/using-audit-logs.md

```diff
@@ -0,0 +1,169 @@
+# Understanding Audit Logs
+
+From time to time, users in "An Idiot's Guide Official Server" need to reference audit logs for whatever reason, let it be viewing them for certain actions to adding things into the audit logs. This guide will explain everything about audit logs and how to use them.
+
+The first thing that you will need is a working Discord Bot. If you do not have one, please visit [Your First Bot](getting-started/your-basic-bot.md) to get started. Now, some things to take note of. This guide is using discord.js@11.3.2. The bot will need some permissions. The main permission the bot will need is `'VIEW_AUDIT_LOGS'`. This permission allows the bot to view the audit logs.
+
+Now that the permission has been established. Lets get started!
+
+Firstly, we need to know what we are doing with the audit logs. 
+Let's log who deleted a message using the messageDelete event. This event will fire whenever a cached message is deleted in a server.
+
+```js
+client.on('messageDelete', async (message) => {
+  // Firstly, we need a logs channel. 
+  const logs = message.guild.channels.find('name', 'logs');
+  
+  // If there is no logs channel, we can create it if we have the 'MANAGE_CHANNELS' permission
+  // Remember, this is completely options. Use to your best judgement.
+  if (message.guild.me.hasPermission('MANAGE_CHANNELS') && !logs) {
+    message.guild.createChannel('logs', 'text');
+  }
+  
+  // If we do not have permissions, console log both errors
+  if (!message.guild.me.hasPermission('MANAGE_CHANNELS') && !logs) { 
+    console.log('The logs channel does not exist and tried to create the channel but I am lacking permissions')
+  }
+  
+  // Now that we have the logs channel created...hopefully...we can send messages to it
+  // before we do that, lets establish who deleted the message
+  // The "type" is how you will be searching through the audit logs. Like roles updated or members banned.
+  // *A complete list of types can be found at the end of this page.*
+  
+  // Define an empty user for now.
+  let user = ""
+  const executor = await message.guild.fetchAuditLogs({type: 'MESSAGE_DELETE'}).then(audit => audit.entries.first().executor.username)
+  
+  
+})
+```
+
+What about bots? Bot messages can be deleted, and they can delete thier own. How can we tell if it is a bot? Great question. The answer is you don't, at least not without some sort of storage data within your own bot. I used a small enmap database that allows me to show that the user was the bot based on where the bot is deleting the messages, whether it was a innapropriate word or a link. 
```

Comment:
Not sure what you're trying to say with this paragraph. Whether it's a bot or not doesn't really change what this particular script does, so it's sort of irrelevant. And you saving what users are a bot is weird, since `user.bot` returns a boolean that tells you this (so executor.bot would give that info).